SegmentEditorMaskVolume/SegmentEditorMaskVolumeLib/SegmentEditorEffect.py

```diff
@@ -44,19 +45,58 @@ def setupOptionsFrame(self):
     self.operationRadioButtons.append(self.fillOutsideButton)
     self.buttonToOperationNameMap[self.fillOutsideButton] = 'FILL_OUTSIDE'
 
+    self.binaryMaskFillButton = qt.QRadioButton("Binary mask")
```

Comment:
Maybe "Fill inside & outside" or  "Fill completely" would be easier to understand.